truffle-config.js

```diff
@@ -23,10 +23,10 @@ const mocha = process.env.GAS_REPORTER ? mochaGasSettings : {}
 module.exports = {
   networks: {
     rpc: {
-      network_id: 15,
+      network_id: 454545,
```

Comment:
why this network id?

Reply:
it is still 15 on `manage-nodes.sh`

Reply:
I think that was left over from me making a private PoA network in geth. I think that it can just stay at 15. Will revert that change.